testcases/broker/kafka-broker/kafka-broker-ordered/100-pod.yaml

```diff
@@ -1,35 +0,0 @@
-apiVersion: v1
```

Comment:
the `kafka-broker-ordered/` folder still has a yaml file

Reply:
See https://github.com/openshift-knative/eventing-hyperfoil-benchmark/pull/27#discussion_r850267486